IntegrationTests/common/script/impl.tcl

```diff
@@ -0,0 +1,31 @@
+#####################
```

Comment:
Would we ever want to run implementation like this? Do we want to run post-implementation timing simulation here? I'm not sure if it really tells us anything accurate. In order to put this on chip we will need to compile this with the actual pin assignments and a top level that connects the actual input/output pins to this 'top level'. When I ran this I saw a lot of "Route 35-198" warnings in the log that complains that "Port XXX does not have an associated HD.PARTPIN_LOCS, which will prevent the partial routing of the signal XXX. Without this partial route, timing analysis to/from this port will not be accurate, and no routing information for this port can be exported." Even if we want to look at the post-implementation timing report for such a simple chain I'm not sure if this provides any accuracy.

Reply:
Everything you say at true. Though presumably, we'll need to run implementation when we run the chain in the Apollo, so even if people don't use this script now, it's perhaps useful to have it there for the future? And it documents the useful Vivado implementation options.

Reply:
Yes we need post-implementation timing report when we run the chain on Apollo. For the baseline FW we already do all the reports here: `https://github.com/apollo-lhc/CM_FPGA_FW/blob/develop/scripts/Build.tcl`.

Reply:
I think it's a complete waste of time to run implementation at this stage. We should aim to make our workflow as simple as possible and not introduce unnecessary steps. I fail to see any justification to put this here. Maybe others can chime in.

Reply:
I'd like to merge this PR soon, so everyone can work with this simplified TB. Is your only remaining concern impl.tcl? I'd like to keep this file, as I think it may be useful in the future. But would it satisfy you if we either: (1) Explain in README.md that running it is not required; or (2) Remove all mention of it from README.md?

Reply:
@rzouCERN -- I've just made mod to README.md agreed at meeting, making clear impl.tcl is optional, and not required for code validation.

Reply:
thanks! Looks good to me.